.github/workflows/ci.yml

```diff
@@ -33,4 +33,4 @@ jobs:
           cd $GITHUB_WORKSPACE/test-message-sender
           mvn -e -X clean install
           cd $GITHUB_WORKSPACE/jpo-conflictmonitor
-          mvn -e -X clean org.jacoco:jacoco-maven-plugin:prepare-agent package sonar:sonar -Dsonar.projectKey=usdot-jpo-ode_jpo-conflictmonitor -Dsonar.projectName=jpo-conflictmonitor -Dsonar.organization=usdot-jpo-ode-1 -Dsonar.host.url=https://sonarcloud.io -Dsonar.branch.name=$GITHUB_REF_NAME
+          mvn -e -X clean org.jacoco:jacoco-maven-plugin:prepare-agent package sonar:sonar -Dsonar.projectKey=usdot-jpo-ode_jpo-conflictmonitor -Dsonar.projectName=jpo-conflictmonitor -Dsonar.organization=usdot-jpo-ode -Dsonar.host.url=https://sonarcloud.io -Dsonar.branch.name=$GITHUB_REF_NAME

```

Comment:
Do we need these arguments and the pom.xml properties. It seems both maybe unnecessary.